packages/ui/src/components/Statistic/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ReactElement } from 'react';
+
+import { TABLE_EMPTY_PLACE_HOLDER } from '../../common/constants';
+import { numberFormat } from '../../utils/numberUtils';
+
+import styles from './index.module.scss';
+
+export type TStatistic = {
+    count?: number;
+    icon: React.ReactNode;
+    label: string;
+};
+
+export const Statistic = ({ count, icon, label }: TStatistic): ReactElement => (
```

Comment:
Works seems great, but I find the component a little bit confusing. Could it be named StatisticEntityIconsList?

Reply:
C'est le nom de Figma je peux le renommer oui mais pourquoi list ?